### ISSUE TITLE
Implement discord-button on startup dialog

### ISSUE THREAD

**Issue by Somebodyisnobody** [legacyclonk/LegacyClonk#44]
Request to implement a button on the startup dialog to enable newbies connecting with the community.

**Comment by maxmitti** [comment]
I think there were already multiple requests to put various links somewhere in the menus.
It would be better to have a thought through solution that links to most useful resources than randomly adding buttons for things one by one.

**Comment by Somebodyisnobody** [comment]
Good idea, in which framework can such a collaborative elaboration be well documented?

**Comment by Fulgen301** [comment]
Here? That's what this issue is for after all.

**Comment by Somebodyisnobody** [comment]
But @maxmitti seems not to be happy with this place when I understood him right

**Comment by maxmitti** [comment]
No, here is ok too. I just wanted a broader scope.
